tests/test_django.py

```diff
@@ -16,3 +22,17 @@ def test_django_landing_page_loads():
 
     assert response.status_code == HTTPStatus.OK
     assert response.json()["title"] == "pygeoapi default instance"
+
+
+def test_django_edr_without_instance_id():
```

Comment:
I've recently added the other django test above and since it was just one test, I didn't add any nice test setup. However for it run, it was necessary to set DJANGO_SETTINGS_MODULE for `django.setup()`.
Here this is now only happening for the test above and not the new one, so how does this work when only the test below is run?

If we need the patching for all tests in this file, we could also do it in some kind of fixture, or maybe have a fixture which returns the django test client (with the correct `SERVER_NAME`) and also does the setup.

Reply:
Good point, I was not quite sure how this was supposed to work. I've made a version with fixtures, please take a look

Reply:
It looks reasonable for our use case. Btw there is a django-pytest module, but i think that's too much for a handful tests, so i'm in favor of this implementation.